src/content/test/text-legibility/guidance.tsx

```diff
@@ -40,17 +57,57 @@ export const guidance = create(({ Markup, Link }) => (
                         ratio (for example, white text on a sky-blue background).
                     </li>
                 </ul>
+
+                <h3>
+                    Make sure users can increase the spacing between letters, words, lines of text, and paragraphs with no clipping or
+                    overlapping. (<Link.WCAG_1_4_12 />)
+                </h3>
+                <ul>
+                    <li> All text must respond to user-initiated changes in spacing.</li>
+                    <li> All text must remain visible, with no clipping or overlapping.</li>
+                </ul>
+
+                <h3>
+                    Make sure users can zoom the browser to 400% and still read the text without having to scroll in two dimensions. (
+                    <Link.WCAG_1_4_10 />)
+                </h3>
+                <ul>
+                    <li> Ideally, scripts read horizontally should require only vertical scrolling, and </li>
```

Comment:
The term "script" here threw me off a bit; I expect the term "script" to mean "a bit of code" or "a `<script>` tag" in a web development context. "Text that is" or "Text in a language that is" maybe?

Reply:
(same for similar guidance in reflow.tsx)

Reply:
I can see what you meen, **Text**  sounds better